opentelemetry collector documentation on monitoring jobs (#3275)
SumoLogic/sumologic-documentation: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md

Make sure you are using collector version `v0.78.0-sumo-0` or higher.

## Collecting logs from other sources

> The full list of receivers available can be found on [Sumo Logic OpenTelemetry Collector page][collector_components_docs]. When the Sumo Logic OpenTelemetry Collector does not support collecting logs from a source, the [Monitoring Job Receiver (Beta)][monitoring_job_receiver_docs] can be used as a catch-all.

Reviewer: ```suggestion
You can find the full list of receivers available on the [Sumo Logic OpenTelemetry Collector][collector_components_docs] page. When the Sumo Logic OpenTelemetry Collector does not support collecting logs from a source, the [Monitoring Job Receiver (Beta)][monitoring_job_receiver_docs] can be used as a catch-all.
```

The Monitoring Job Receiver can be configured to execute a script or command on the collector host. The standard output and error streams from that command will be collected as log record(s).

> :::important  Prefer purpose-built receivers to Monitoring Jobs.

Reviewer: ```suggestion
:::important Prefer purpose-built receivers to Monitoring Jobs
```

When available, prefer purpose-built receivers like `sqlquery` to monitoring jobs receiver. This example monitoring job configuration runs SQL queries using the `psql` tool for demonstration purposes as it is well understood.
:::

> This configuration configures the controller to execute the `psql` command and collects the output text as logs. See the [monitoring job documentation][monitoring_job_receiver_docs] for configuration specifics including advanced output processing with features like multi-line log detection, timestamp and field parsing.

Reviewer: ```suggestion
This configuration configures the controller to execute the `psql` command and collects the output text as logs. See the [Monitoring Job Receiver documentation][monitoring_job_receiver_docs] for configuration specifics including advanced output processing with features like multi-line log detection, timestamp and field parsing.
```
